v-next/hardhat/templates/mocha-ethers/test/Lock.ts

```diff
@@ -0,0 +1,156 @@
+import hre from "@ignored/hardhat-vnext";
+import { NetworkConnection } from "@ignored/hardhat-vnext/types/network";
+
+import "@ignored/hardhat-vnext-ethers";
+import "@ignored/hardhat-vnext-network-helpers";
+
+// TODO: Chai as promised support needs to be added either directly
+// or through the updated `chai-matchers` package.
+import * as chai from "chai";
+import chaiAsPromised from "chai-as-promised";
+chai.use(chaiAsPromised);
+const { expect } = chai;
```

Comment:
I applied the `chaiAsPromised` directly here.
I don't think we currently have support `.mochars.json` files to allow us to setup up supporting setup files.